authentication/pipeline/user.py

```diff
@@ -16,9 +18,14 @@
 from authentication.utils import SocialAuthState
 
 from compliance import api as compliance_api
+from courseware import api as courseware_api, tasks as courseware_tasks
 from hubspot.task_helpers import sync_hubspot_user
 from users.serializers import UserSerializer, ProfileSerializer
 
+log = logging.getLogger()
+
+CREATE_COURSEWARE_USER_RETRY_DELAY = 60
```

Comment:
nit/question - Would this be better if it were defined as a Heroku config?

Reply:
Probably, but for the sake of getting this PR out faster and not having to write out app.json, and tests for this, I'd defer it for a tech debt pr.